Persistence/EntityTypeConfigurations/WineConfiguration.cs

```diff
@@ -15,8 +15,9 @@ public void Configure(EntityTypeBuilder<Wine> builder)
         builder.HasIndex(e => e.Type);
         builder.Property(e => e.Type).HasMaxLength(50);
         builder.HasIndex(e => e.Quantity);
-        builder.HasIndex(e => e.UserId);
         builder.HasIndex(e => e.Description);
         builder.Property(e => e.Description).HasMaxLength(250);
+        builder.HasIndex(e => e.ExpirationDate);
+        builder.HasOne(e => e.Storage).WithMany(e => e.Wines).HasForeignKey(e => e.StorageId);
```

Comment:
i har alt for mange index kolonner. Gennemtaenkt og se hvad implikationer der ved ved det

Reply:
thumbs up